### PR TITLE
Removes airlock charges.

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

Removes airlock charges from the code.
Gives the sabotage kit some more plastic explosives, a box of signalers to compensate and a duffelbag to compensate.

## Why It's Good For The Game

Airlock charges are bad. I mean gameplay wise. We already had that discussion long ago (#27614) yet the shitcode still remains. This is basically just clean up.

## Changelog
:cl:
add: Sabotage bundle had their airlock charges removed. As compensation they gets some more plastic explosives and a box of signalers.
del: Removed airlock charge from the game
/:cl:

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
a box in a dufflebag in a box in a bag

**Comment by actioninja** [comment]
Airlock charges suck hard, good change.

**Comment by Shadowflame909** [comment]
this item already got replaced by the explosive holoparasite anyways

**Comment by nemvar** [comment]
Yes. this item is not worth the shitcode associated with it.

File: code/game/objects/items/storage/uplink_kits.dm [hunk 1]
```diff
@@ -90,14 +90,11 @@
 			new /obj/item/storage/toolbox/syndicate(src)
 
 		if("sabotage")
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/storage/uplink_kits.dm]: is the sabotage bag suppose to be in here? You removed 2 C4 otherwise.

Comment (nemvar) [reply code/game/objects/items/storage/uplink_kits.dm]: Messing up the easiest things. smh

**Pull request merged**